docs/plugin/interfaces/tags.md

```diff
@@ -106,6 +106,8 @@ Ddefines a literal as a known identifier (equivalent of `#define` in C++ code)
 |[@interface](#interface)| Specifies a parameter holding interface ID value for void* interface passing |  | Yes | No |Method paramter|
 |[@length](#length)|Specifies the expression to evaluate length of an array parameter (can be other parameter name, or constant, or math expression)|  | No | Yes | Method Parameter|
 |[@maxlength](#maxlength)|Specifies a maximum buffer length value |  | No | Yes |Method parameter|
+|[@default](#default)|Provides a default value for an unset optional type |  | Yes | Yes |Method parameter|
+|[@encode:base64](#encode:base64)|Encodes C-Style arrays as JSON-RPC base64 arrays |  | Yes | Yes |Method parameter|
```

Comment:
To be in line with the above documentation think the base64 should not be here but just document #encode influences the generated encoding and list the options (so also bitmask, see Sebastians presentation ) where the details are documented (so more like @text is documented). And isn't it only for json rpc so should it not be in the table below? And previously in the variable size arrays I'm sure it generated a string, so perhaps that is still the case?